packages/driver/src/cypress/error_messages.coffee

```diff
@@ -109,7 +109,7 @@ module.exports = {
     }
 
   browser:
-    invalid_arg: "`Cypress.{{method}}()` must be passed the name of a browser or an object to filter with. You passed: `{{obj}}`"
+    invalid_arg: "{{prefix}} must be passed the name of a browser or an object to filter with. You passed: `{{obj}}`"
```

Comment:
Doesn't this also accept an array though? And an 'exclusion' matcher (which is not a `name` of the browser). I'm confused by this example in the ts definitions `@example isBrowser(['firefox', 'edge'])` is this valid?

Reply:
updated